docs/docs/ref-05-events.ko-KR.md

```diff
@@ -139,7 +139,6 @@ DOMEventTarget relatedTarget
 
 이 포커스 이벤트는 폼 엘리먼트뿐만 아니라 모든 React DOM 엘리먼트에서 작동합니다.
 
-<a name="form-events"></a>
```

Comment:
Since [kramdown](https://github.com/facebook/react/blob/master/docs/Gemfile.lock#L43) strip i18n characters.

> http://facebook.github.io/react/docs/events-ko-KR.html#폼-이벤트

is not valid link.
Auto-generated link (`http://facebook.github.io/react/docs/events-ko-KR.html#-`) is pointing "이벤트 풀링".

You can check this behaviour on current page. 
http://facebook.github.io/react/docs/events-ko-KR.html#-